clusterloader2/testing/load/experimental/extended_config.yaml

```diff
@@ -126,8 +134,18 @@ steps:
       kind: Deployment
       labelSelector: group = load
       operationTimeout: 15m
+  {{if $ENABLE_STATEFULSETS}}
+  - Identifier: WaitForRunningStatefulSets
+    Method: WaitForControlledPodsRunning
+    Params:
+      action: start
+      apiVersion: apps/v1
+      kind: StatefulSet
+      labelSelector: group = statefulset
```

Comment:
This one is pretty much only selecting `controlling objects` (so statefulsets).

So for consistency with deployments (and probably other in the near future), I vote for making that:
```
group = load
```

This would mean that they will also be measured as part of pod-startup-time and all stuff that was assuming that all pods actually have group = load label.

WDYT?